docs/csharp/language-reference/compiler-options/langversion-compiler-option.md

```diff
@@ -47,15 +47,21 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
   
 |Option|Meaning|  
 |------------|-------------|  
-|default|The compiler accepts all valid language syntax.|  
-|ISO-1|The compiler accepts only syntax that is included in the ISO/IEC 23270:2003 C# language specification.|  
-|ISO-2|The compiler accepts only syntax that is included in the ISO/IEC 23270:2006 C# language specification. This specification is available on the [ISO](http://go.microsoft.com/fwlink/?LinkId=144406) Web site.|  
-|3|The compiler accepts only syntax that is included in the version 3.0 [C# Language Specification](../../../csharp/language-reference/language-specification.md).|  
+|default|The compiler accepts all valid language syntax that it can support.|  
+|ISO-1|The compiler accepts only syntax that is included in the ISO/IEC 23270:2003 C# language specification. (net 2.x or newer)|  
+|ISO-2|The compiler accepts only syntax that is included in the ISO/IEC 23270:2006 C# language specification. This specification is available on the [ISO](http://go.microsoft.com/fwlink/? LinkId=144406) Web site. (net 3.x or newer)|
```

Comment:
It seems you broke the link to the ISO spec by accident by adding a space after the `?`.